test/download.js

```diff
@@ -62,6 +62,63 @@ test('download: Download with default opts', function (t) {
   })
 })
 
+test('download: Download with secretDir opt', function (t) {
```

Comment:
Oh! Sorry was not looking closely at this before, we only have a secret directory when you create the Dat (e.g. sharing). 

These tests look good otherwise. Can you also just make sure the `downSecretDir` has a key in it?